macros/edr/materializations/tests/test.sql

```diff
@@ -1,3 +1,16 @@
+{% macro create_test_result_temp_table() %}
+  {% set database, schema = elementary.get_package_database_and_schema() %}
+  {% set relation = make_temp_relation(api.Relation.create(identifier=model["alias"], database=database, schema=schema, type="table")) %}
+  {% call statement("temp_table") %}
+    {{ create_table_as(True, relation, sql) }}
+  {% endcall %}
+  {% set new_sql %}
+    select * from {{ relation }}
+  {% endset %}
+  {% do context.update({"sql": new_sql}) %}
```

Comment:
This is a side-effect for this function, it's not clear that it overrides the `sql` context.
I think we should do that after the call to `create_test_result_temp_table` in the caller function.

Reply:
👍